src/test/java/com/google/jenkins/plugins/computeengine/InstanceConfigurationTest.java

```diff
@@ -184,68 +187,75 @@ public void testConfigRoundtrip() throws Exception {
   public void testInstanceModel() throws Exception {
     Instance i = instanceConfiguration(MIN_CPU_PLATFORM).instance();
     // General
-    assert (i.getName().startsWith(NAME_PREFIX));
-    assert (i.getDescription().equals(CONFIG_DESC));
-    assert (i.getZone().equals(ZONE));
-    assert (i.getMachineType().equals(MACHINE_TYPE));
-    assert (i.getMinCpuPlatform().equals(MIN_CPU_PLATFORM));
+    assertTrue(i.getName().startsWith(NAME_PREFIX));
+    assertTrue(i.getDescription().equals(CONFIG_DESC));
```

Comment:
Like the cleanup of these asserts. Recommend `assertEquals` here and everywhere else a `.equals()` is wrapped in an `assertTrue` call

Reply:
*almost everywhere as I found while looking through the rest

Reply:
Done.